manifests/install.pp

```diff
@@ -40,9 +40,9 @@
 
   if !$::vault::disable_mlock {
     exec { "setcap cap_ipc_lock=+ep ${vault_bin}":
```

Comment:
Commits are fine, there's not too many.

Chances are you weren't setting the right env vars for the tests, e.g.:

```
env STRICT_VARIABLES="no" bundle exec rake spec
```